projects/packages/external-connections/src/class-external-connections.php

```diff
@@ -0,0 +1,299 @@
+<?php
+/**
+ * Management of external connections.
+ *
+ * @package automattic/jetpack-external-connections
+ */
+
+namespace Automattic\Jetpack;
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+use Automattic\Jetpack\Status\Host;
+
+/**
+ * Main class.
+ */
+class External_Connections {
+
+	const PACKAGE_VERSION = '0.1.0-alpha';
+	const BASE_FILE       = __FILE__;
+
+	/**
+	 * List of services whose connections are managed in settings pages.
+	 *
+	 * Each item has a key with the slug of the settings page, and a value with an array of services.
+	 *
+	 * Each service has the following keys:
+	 * - service: The service identifier.
+	 * - title: The title of the service.
+	 * - description: The description of the service.
+	 * - support_link: An array with the following keys:
+	 *     - jetpack: The URL handler registered in jetpack.com/redirect/.
+	 *     - wpcom: The URL of the support page for the service on WordPress.com.
+	 *
+	 * @example
+	 * ```php
+	 * $this->services = array(
+	 *     'media' => array(
+	 *         array(
+	 *             'service'      => 'facebook',
+	 *             'title'        => 'Facebook',
+	 *             'description'  => 'Connect your site to your Facebook account',
+	 *             'support_link' => array(
+	 *                 'jetpack' => 'facebook-connection',
+	 *                 'wpcom'   => 'https://wordpress.com/support/facebook/',
+	 *             ),
+	 *         ),
+	 *     ),
+	 * );
+	 * ```
+	 * @var array
+	 */
+	private static $services = array();
+
+	/**
+	 * Gets the connect URL for a given service.
+	 *
+	 * @param string $service The service identifier.
+	 * @return string|null The connect URL, or `null` if the service is not supported.
+	 */
+	public static function get_connect_url( $service ) {
+		if ( ( new Host() )->is_wpcom_simple() ) {
+			require_lib( 'external-connections' );
+			$connections = \WPCOM_External_Connections::init();
+			$service     = $connections->get_external_service_item( $service );
+			return empty( $service ) ? null : $service['connect_URL'];
```

Comment:
Nitpick, but there's no need for the `empty()` here; it always either returns an array with the proper keys or an empty array (though the PHPDoc for `get_external_service_item()` says it returns `false`). You could do this:


```suggestion
			return ! $service ? null : $service['connect_URL'];
```

Or perhaps more readable:
```suggestion
			return $service ? $service['connect_URL'] : null;
```

Or even this:
```suggestion
			return $service['connect_URL'] ?? null;
```

Reply:
`get_external_service_item` can indeed return `false` (see fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Syvo%2Srkgreany%2Qpbaarpgvbaf.cuc%3Se%3Q8455q799%23529-og), so I don't think we can use the null coalescing operator. 

I ended up changing it to `$service ? $service['connect_URL'] : null;` in https://github.com/Automattic/jetpack/pull/44858/commits/fb06edd71b565a3a78938afe8eb567219894f0d4

Reply:
Oops, I was looking at `get_external_services_list`! Thanks for verifying! 😅